api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -64,4 +64,10 @@ message RateLimit {
   // success.
   config.ratelimit.v3.RateLimitServiceConfig rate_limit_service = 7
       [(validate.rules).message = {required: true}];
+
+  // If set to true enables emitting X-RateLimit-* headers by the filter.
```

Comment:
To avoid having to read the RFC in all cases, can you spell out a bit more what headers are emitted and what they do?